pages/api/auth/[...nextauth].tsx

```diff
@@ -76,15 +76,34 @@ export default NextAuth({
           username: user.username,
           email: user.email,
           name: user.name,
+          locale: user.locale,
         };
       },
     }),
   ],
   callbacks: {
     async jwt(token, user) {
+      // If we update the user username/locale
+      // the token is not updated with the
+      // updated username/locale (only after signing in again)
+      // If session exists, update with the right value
+      if (!user && token.id) {
+        const currentUser = await prisma.user.findUnique({
+          where: {
+            id: token.id,
+          },
+        });
+        if (currentUser) {
+          token.id = currentUser.id;
+          token.username = currentUser.username;
+          token.locale = currentUser.locale;
```

Comment:
Actuallyyyyy --- I think I found a potential issue?

From what I can tell the `locale` should live on the locale _or_ in the DB - not both?

Reply:
I meant "in the **jwt** _or_ in the DB"